### PR TITLE
Add a review success page

### PR DESCRIPTION
**Pull request by ghickman**
This fleshes out the submission of a review to add a final page where a user can download the approved outputs and a summary of the review for use elsewhere.

**Review complete page:**

Screenshot 2023-07-10 111128

---

**Summary document:**

Fix: #149 
Fix: #139

### CODE REVIEW

File: sacro/views.py [hunk 1]
```diff
@@ -199,6 +201,28 @@ def review_detail(request, pk):
...
         "counts": counts,
         "review": review,
+        "summary_url": summary_url,
     }
 
     return TemplateResponse(request, "review.html", context=context)
+
+
+@require_POST
+def summary(request, pk):
```
Comment (lucyb) [inline sacro/views.py:211]: Why is this a POST rather than a GET?

File: sacro/templates/index.html [hunk 2]
```diff
@@ -139,7 +139,7 @@ <h1 class="text-xl font-semibold leading-7 text-gray-900" data-sacro-el="outputT
   </main>
 
   {% #modal id="submitModal" title="Release and download" subtitle='<span class="count"></span> files are ready to be released' %}
-    <form action="{{ submission_url }}" method="post" id="approveForm" class="flex flex-col gap-3">
+    <form action="{{ create_url }}" method="post" id="approveForm" class="flex flex-col gap-3">
```
Comment (lucyb) [inline sacro/templates/index.html:142]: `create_url` sounds a bit vague. What are we creating? Is it possible to be more specific here?

File: cypress/e2e/spec.cy.js [hunk 3]
```diff
@@ -11,11 +13,20 @@ describe("Approve all files and download ZIP", () => {
...
+    cy.get("#approveForm").submit();
+
+    cy.url().should("eq", "http://localhost:8000/review/current/");
+    //
     // prepare for form submission that returns back a file
     // https://on.cypress.io/intercept
     cy.intercept(
       {
-        pathname: "/review*",
+        pathname: "/review/current/approved-outputs/",
```
Comment (lucyb) [inline cypress/e2e/spec.cy.js:29]: Might be good to do something similar here for downloading the review summary and checking that a text file is returned, assuming that's quite straightforward to do.

**Review by lucyb — APPROVED** [review]

File: sacro/templates/index.html (hunk 2 above)
Comment (ghickman) [reply sacro/templates/index.html:142]: Reviews… the only thing we create in the entire Django app 😂

File: sacro/views.py (hunk 1 above)
Comment (ghickman) [reply sacro/views.py:211]: IIRC we have to POST to get a file save dialogue without the page changing when someone clicks the download button.  I mirrored the approved_outputs view for this.

File: cypress/e2e/spec.cy.js (hunk 3 above)
Comment (ghickman) [reply cypress/e2e/spec.cy.js:29]: Done.  I wasn't sure how to simplify the `oneOf` for a single item so I've left it as is.

**Pull request merged**